src/hooks/useSnackMessage.ts

```diff
@@ -97,18 +117,18 @@ function checkAndTranslateIfNecessary(txt, id, values, intlRef) {
     );
 }
 
-function checkInputs(txt, id, values) {
+function checkInputs(txt?: string, id?: string, values?: any) {
     if (txt && (id || values)) {
         console.warn('Snack inputs should be [*Txt] OR [*Id, *Values]');
     }
 }
 
 function displayMessageWithSnackbar(
-    message,
-    header,
-    enqueueSnackbar,
-    level,
-    persistent
+    message: string,
+    header: any,
+    enqueueSnackbar: any,
+    level: any,
+    persistent: any
```

Comment:
We knows the types

Reply:
Specified it as follow 

>     header: string,
>     enqueueSnackbar: EnqueueSnackbar,
>     level: 'default' | 'error' | 'success' | 'warning' | 'info' | undefined,
>     persistent: boolean

Reply:
Instead of duplicating the level type, just use the one provided by notistack
`type BaseVariant = 'default' | 'error' | 'success' | 'warning' | 'info';`